lib/consensus/isaac.go

```diff
@@ -88,7 +88,7 @@ func (is *ISAAC) CloseConsensus(proposer string, round round.Round, vh ballot.Vo
 	return
 }
 
-func (is *ISAAC) SetLatestConsensusedBlock(block block.Block) {
+func (is *ISAAC) SetLatestConfirmedBlock(block block.Block) {
```

Comment:
That's already a much better name, but what about just "SetLatestBlock" ?

Reply:
This will be handled in #399 :)